app/components/sidebars/main/channels_list/filtered_list/index.js

```diff
@@ -112,7 +113,8 @@ function mapStateToProps(state) {
         currentChannel: getCurrentChannel(state),
         currentTeam: getCurrentTeam(state),
         currentUserId,
-        otherChannels: getOtherChannels(state),
+        otherChannels: getOtherUnArchivedChannels(state),
```

Comment:
can't we just pass an argument to the existing selector to include or not archived channels?

Reply:
done. updated redux pr as well